sopht/utils/restart_sim.py

```diff
@@ -0,0 +1,46 @@
+from typing import Type, Union
```

Comment:
`Union` may be unneeded. See the following comment. 
```suggestion
from typing import Type
```